pmg/views.py

```diff
@@ -252,8 +274,8 @@ def get_month_unicode(m_date):
 
         filtered_meetings[meeting_year].append(meeting)
 
-    latest_year = max(y for y in filtered_meetings)
-    earliest_year = min(y for y in filtered_meetings)
+    latest_year = max(y for y in filtered_meetings) if len(filtered_meetings) else None
```

Comment:
In python, `if filtered_meetings` is generally how this test is done. Both None and an empty list are falsey.